src/Common/sendRequest.cpp

```diff
@@ -33,10 +33,10 @@ std::pair<String, Int32> sendRequest(
     const String & password,
     const String & payload,
     const std::vector<std::pair<String, String>> & headers,
-    Poco::Logger * log)
-{
+    Poco::Logger * log,
     /// One second for connect/send/receive
-    ConnectionTimeouts timeouts({2, 0}, {5, 0}, {10, 0});
+    ConnectionTimeouts timeouts)
```

Comment:
the last param is 'log', move the 'timeouts' param up.

Reply:
`timeouts` is orginally a local variable in this function. This change will maximize compatibility for other codes using `sendRequest`

Reply:
I have moved the `timeout` param up.